delivery_carrier_pricelist/models/stock_picking.py

```diff
@@ -13,7 +13,7 @@ def _get_view(self, view_id=None, view_type="form", **options):
         arch, view = super()._get_view(
             view_id=view_id, view_type=view_type, options=options
         )
-        if view.name == "stock.picking.form":
+        if view.name == "delivery.stock.picking_withcarrier.form.view":
```

Comment:
why? `stock.picking.form` has been renamed?

Reply:
in version 16.0, when `delivery` depends `sale_stock`, `stock.picking.form` have fields such as `//button[@name='cancel_shipment']`, `//button[@name='send_to_shipper']`, `//field[@name='partner_id']`.

now, they separated `delivery` to `stock_delivery`. these field above is in [delivery.stock.picking_withcarrier.form.view](https://github.com/odoo/odoo/blob/fa9d12d1a9899afff5660fd1bdb77e4523c8cacd/addons/stock_delivery/views/delivery_view.xml#L28)

Reply:
The author's original intention was to inherit the original view so he input `"stock.picking.form"`, and in fact since version 14.0 (according to my check) the above buttons always exist in the view `"delivery.stock.picking_withcarrier.form.view"`, but in every migration process no one changes it.
can you put it back and test, does it really still work? i might be missing something, but be patient and try again